functions/src/create-answer.ts

```diff
@@ -124,12 +126,6 @@ export const createAnswer = functions.runWith({ minInstances: 1 }).https.onCall(
         volume: volume + amount,
       })
 
-      if (!isFinite(newBalance)) {
```

Comment:
I guess it's fine to remove the sanity check if amount is suitably validated.

Reply:
The `zod` `number` validator enforces finitude.